### PR TITLE
Micro Reddit

### PR DESCRIPTION
**Pull request by jcy2704**
### Micro Reddit Active Record Project
*We added different models with their validations that composed basic features of Reddit which are:*
- **User**
  - _username_
  - _password_
  - _email_
- **Post**
  - _title_
  - _body_
- **Comment**
  - _comment_

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this milestone :+1:__

- Your project passed all the test cases :tada:

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] **Your project do not have a descriptive README.md** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- I believe your mind skipped this important requirement. Kindly update your README.md file. 


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Good job :+1: This project Approved :white_check_mark:

#### _Reviewed by Temtayo Adeleke_

**Pull request merged**
